source/_data/themes.yml

```diff
@@ -2879,3 +2879,17 @@
     - rem
     - simple
     - pure
+- name: hexo-theme-clean-dark
```

Comment:
```suggestion
- name: clean-dark
```

Reply:
Hi, I'd renamed my screenshot to clean-dark.png